Makefile

```
@@ -4,19 +4,32 @@ NON_MATCHING ?= 0
 # libgultra_rom, libgultra_d, libgultra
 # libultra_rom, libultra_d, libultra
 TARGET ?= libgultra_rom
+VERSION ?= K
```

```suggestion
VERSION ?= L
```
Should probably stay defaulted to latest version?

---

Fixed